scripts/docs/metalsmith/content/concepts.md

```diff
@@ -46,8 +59,8 @@ input the string parameters, but this interface (that we call 'raw')
 is exclusive with the higher level one (that we call 'managed').
 
 This layer on top of the Algolia API even let us create features that don't
-exist in the normal API. We have created some features that rely on multiple
-queries based on the pattern we found over time.
+exist in the raw Algolia API. We have created some features that rely on multiple
```

Comment:
I tried to use raw only for the parameters and configuration inside the helper, to avoid confusions. What about using Rest API here?